sdcm/cluster.py

```diff
@@ -1872,10 +1874,43 @@ def run_nodetool(self, sub_cmd, args="", options="", ignore_status=False):
         """
         cmd = self._gen_nodetool_cmd(sub_cmd, args, options)
         result = self.remoter.run(cmd, ignore_status=ignore_status)
-        self.log.debug("Command '%s' duration -> %s s", result.command,
-                       result.duration)
+        self.log.debug("Command '%s' duration -> %s s" % (result.command, result.duration))
+
+        if sub_cmd == 'status':
+            try:
+                self.validate_schema_version()
+            except Exception as e:
+                InfoEvent('Validate schema version failed. Error: {}'.format(e.message))
         return result
 
+    def validate_schema_version(self):
+        # Get schema version
+        cmd = self._gen_nodetool_cmd('gossipinfo', args='', options='')
```

Comment:
What is the reason that you generate the nodetool command instead of using `self.run_nodetool("gossipinfo")`?

Reply:
fixed